wear/src/main/java/chat/rocket/android/util/ThreadUtils.kt

```diff
@@ -0,0 +1,29 @@
+package chat.rocket.android.util
+
+import android.app.Activity
+import android.app.Fragment
+import android.os.Looper
+import kotlinx.coroutines.experimental.Job
+import kotlinx.coroutines.experimental.android.UI
+import kotlinx.coroutines.experimental.launch
+
```

Comment:
What about putting this class in the `util` module instead in the `wear` module?.
Also, there is a unneeded blank space here.